pydropsonde/helper/__init__.py

```diff
@@ -263,12 +267,17 @@ def calc_q_from_rh(ds):
     )
 
     q = typhon.physics.vmr2specific_humidity(vmr)
-    ds = ds.assign(q=(ds.ta.dims, q))
-    ds["q"].attrs = dict(
-        standard_name="specific humidity",
-        long_name="specific humidity",
-        units="kg/kg ",
+    ds = ds.assign(
+        q=(
+            ds.ta.dims,
+            q,
+            dict(
+                standard_name="specific_humidity",
+                units="1",
+            ),
+        )
     )
+
```

Comment:
yes and no. we calculate it only once in the pipeline currently, but there are two functions to get q from rh, because the correct calculation from the rh we get from the sondes to q is different from the reverse of "our" q-to-rh function. so I wanted to add the reverse function for that as well; just in case someone wants to reverse calculate q from rh in the interpolated dataset